templates/slurmdbd.conf.j2

```diff
@@ -1,14 +1,58 @@
 # Managed by Ansible
 # See the slurmdbd.conf man page for more information.
 #
-# Archive info
-ArchiveJobs=yes
```

Comment:
Based on https://slurm.schedmd.com/slurmdbd.conf.html this changes the default ArchiveJobs of this role (which is ArchiveJobs=yes) to the default of slurmdbd.conf (which is no).

Is this acceptable?

Reply:
I think so, because if I understand that man page correctly, ArchiveJobs matters only if PurgeJobs is set, which wasn't even possible to set previously.